sorc/chgres_cube.fd/write_data.F90

```diff
@@ -55,8 +54,7 @@ subroutine write_fv3_atm_header_netcdf(localpet)
 
  print*,"- WRITE ATMOSPHERIC HEADER FILE: ", trim(outfile)
 
- error = nf90_create(outfile, IOR(NF90_NETCDF4,NF90_CLASSIC_MODEL), &
-                     ncid, initialsize=initial, chunksize=fsize)
+ error = nf90_create(outfile, NF90_NETCDF4, ncid)
```

Comment:
This change loses the NF90_CLASSIC_MODEL flag, and that's fine.

The CLASSIC_MODEL flag ensures that the created file follows netCDF classic rules, so it can be processed by netcdf-classic (i.e. pre-netCDF4) code. Although the CLASSIC_MODEL enforces some restrictions on the netCDF-4 file, it is not really necessary.

If we refrain from using netCDF-4 user-defined types, hierarchical groups, and unsigned integer types (and this is the current plan, AFAIK), then the output files will continue to be readable by netCDF classic code. The NC4_CLASSIC_MODEL flag is not needed for this, we will still get it.

Eliminating the initialsize and chunksize optional parameters is good. They are ignored for NC_NETCDF4 files. NetCDF-4 chunksizes are set on a per-variable basis, when the variable is defined. If you don't explicitly set the variable chunksizes, then netCDF will take a guess and come up with chunksizes for each var.

So this is a good change.